src/mame/bfm/bfcobra.cpp

```diff
@@ -2809,20 +2809,35 @@ void bfcobjam_state::bfcobjam_with_dmd(machine_config &config)
 
   Note: Two different versions of each 6809 ROM exist: standard and protocol
   It appears sets can be a combination of disk images, 6809 and Z80 ROMs!
+  
+  I have just dumped the roms from a board with the labels "95760002" and "viper".
+  The "viper" rom is the same as 9576002.bin below and 95760002 is the same
+  as inqvypp2!
+  The labels are original and don't appear to have been removed from the
+  chips before.
+  I'm unsure where 7 digit prom numbering would've come from. BFM always
+  used 8 digit numbers.
```

Comment:
"I have just" will become outdated the very moment this PR hits master branch. Also you may as well credit yourself on this block of note: the part where this very driver has "Anonymous" as copyright holder won't be helpful in the long run.

Reply:
OK, I'll add my name to that part of the comment block.

As the anonymous copyright holder was also me I'll change that too.